velox/connectors/Connector.h

```diff
@@ -98,6 +102,10 @@ class DataSink {
   virtual void appendData(VectorPtr input) = 0;
 
   virtual void close() = 0;
+
+  // Get all parameters used by the underlying writers
```

Comment:
Would it make sense to move this method before 'appendData'? When is it called in the overall workflow?

Reply:
Done. It's returned to getOutputs() of the TableWriter, wrapped into WriteInfo and passed to WriteProtocol::commit(), since it contains the write & target paths. commit() needs it, and, in the case of Presto, WriteInfo also is eventually serialized to json and sent back to the coordinator for Metastore commit